packages/cxl-ui/src/components/cxl-icon-nav.js

```diff
@@ -0,0 +1,24 @@
+// Import the LitElement base class and html helper function
+import { LitElement, html } from 'lit-element';
+import cxlThemeStyles from '../styles/cxl-theme-css.js';
+import cxlIconNavStyles from '../styles/cxl-icon-nav-css.js';
+
+class CXLIconNavElement extends LitElement {
+  static get properties() {
+    return {};
+  }
+
+  static get styles() {
+    return [cxlThemeStyles, cxlIconNavStyles];
+  }
+
+  render() {
+    return html`
+      <ul>
```

Comment:
`ul`? https://www.w3.org/TR/html50/grouping-content.html#the-ul-element says only `li`, `script`, or `template` elements are allowed in `ul`.

Reply:
If need a container, wouldn't it be a `<nav>`?